Viper/Viper/HotelOfCountry/HotelTableViewCell.swift

```diff
@@ -0,0 +1,81 @@
+// HotelTableViewCell.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Кастомная ячейка отеля
+final class HotelTableViewCell: UITableViewCell {
+    // MARK: - Private visual components
+
+    private let hotelImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.layer.borderColor = UIColor.black.cgColor
+        imageView.layer.borderWidth = 1
+        imageView.layer.cornerRadius = 20
+        imageView.layer.masksToBounds = true
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        return imageView
+    }()
+
+    private let hotelNameLabel: UILabel = {
+        var label = UILabel()
+        label.font = .systemFont(ofSize: 20, weight: .bold)
+        label.textColor = .black
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+
+    private let hotelTextView: UITextView = {
+        let textView = UITextView()
+        textView.font = .systemFont(ofSize: 15, weight: .thin)
+        textView.textColor = .black
+        textView.translatesAutoresizingMaskIntoConstraints = false
+        return textView
+    }()
+
+    // MARK: - Init
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        addSubview(hotelImageView)
+        addSubview(hotelNameLabel)
+        addSubview(hotelTextView)
+        setConstraints()
```

Comment:
```suggestion
        addSubview(hotelImageView)
        addSubview(hotelNameLabel)
        addSubview(hotelTextView)
        setConstraints()
```

вынеси в отдельный метод